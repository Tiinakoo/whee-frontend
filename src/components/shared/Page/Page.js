import React from "react";
import styles from "./Page.module.scss";
import classNames from "classnames";

const Page = ({ className, children, ...props }) => {
  const classes = classNames([styles.page, className]);

  return (
    <div className={classes} {...props}>
      {children}
    </div>
  );
};

export default Page;
