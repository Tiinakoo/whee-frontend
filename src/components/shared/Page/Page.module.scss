@import "../../../styles/baseline";
@import "../../../styles/mixins";

.page {
  max-width: 100%;
}
