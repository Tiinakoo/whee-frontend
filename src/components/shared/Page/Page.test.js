import React from "react";

import { shallow } from "enzyme";
import Page from "./Page";

describe("Page", () => {
  it("renders", () => {
    const component = shallow(<Page />);

    expect(component).toMatchSnapshot();
  });
});
