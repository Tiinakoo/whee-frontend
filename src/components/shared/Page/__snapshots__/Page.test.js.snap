// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page renders 1`] = `
<div
  className="page"
/>
`;
